### PR TITLE
Update to the latest Microsoft.DotNet.Interactive

### PR DESCRIPTION
**Pull request by eerhardt**
This allows the DataFrame extension to work with the latest .NET Interactive.

I also needed to change the DataFrame .csproj so it gets packed by including the right Pack.props file. 

Added an .editorconfig entry so .csproj files are uniform in their spacing.

The only meaningful code change is the rows per page going from 10 to 25. 10 is just too small IMO.

### CODE REVIEW

**Review by pgovind — APPROVED** [review]
LGTM!

**Comment by michaelgsharp** [comment]
It can't find the version of System.CommandLine specified. Do we need a new feed?

**Comment by eerhardt** [comment]
> Do we need a new feed?

Yep, :( added.

**Comment by eerhardt** [comment]
@pgovind @michaelgsharp - this should be green now if you want to take a look.

File: test/Microsoft.Data.Analysis.Interactive.Tests/Microsoft.Data.Analysis.Interactive.Tests.csproj [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
-        <ProjectReference Include="..\..\src\Microsoft.Data.Analysis.Interactive\Microsoft.Data.Analysis.Interactive.csproj" />
-    </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Microsoft.Data.Analysis.Interactive\Microsoft.Data.Analysis.Interactive.csproj" />
 
-    <!-- register for test discovery in Visual Studio -->
-    <ItemGroup>
-        <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
-    </ItemGroup>
+    <!-- work around https://github.com/dotnet/runtime/issues/45560 until System.Text.Json is fixed and Microsoft.DotNet.Interactive updates to the new version -->
```
Comment (pgovind) [inline test/Microsoft.Data.Analysis.Interactive.Tests/Microsoft.Data.Analysis.Interactive.Tests.csproj:10]: This looks fine to me. 

I also just discovered that there's both https://www.nuget.org/packages/Microsoft.DotNet.Interactive/1.0.0-beta.21155.3 and https://www.nuget.org/packages/Microsoft.dotnet-interactive/

**Review by pgovind — APPROVED** [review]

File: test/Microsoft.Data.Analysis.Interactive.Tests/Microsoft.Data.Analysis.Interactive.Tests.csproj (hunk 1 above)
Comment (eerhardt) [reply test/Microsoft.Data.Analysis.Interactive.Tests/Microsoft.Data.Analysis.Interactive.Tests.csproj:10]: Microsoft.dotnet-interactive is the application / global tool. Microsoft.DotNet.Interactive is the API / library.

**Review by michaelgsharp — APPROVED** [review]
LGTM.

**Pull request merged**